src/pages/documentation/consent-for-edge-network/api-reference.md

```diff
@@ -65,4 +65,36 @@ iOS
 
 <Tabs query="platform=ios&api=update-consents"/>
 
+<InlineAlert variant="info" slots="text1, text2, text3, text4, text5"/>
+
+When `Consent.update` API is called, the Consent extension uses Adobe Standard 2.0 to communicate with the Edge Network.  Additionally, the property `metadata` is set to the time at which the API is called.
+
+Here is an example where `Consent.update` is called to set collect consent to y:
```

Comment:
I'd phrase it more as "The following example shows when `Consent.update` is called to set collect consent to `y`:"